.github/workflows/buildAndTest.yml

```diff
@@ -65,6 +65,8 @@ jobs:
           cmake .. \
             -GNinja \
             -DCMAKE_BUILD_TYPE=Debug \
+            -DAIE_COMPILER=NONE \
+            -DAIE_LINKER=NONE \
```

Comment:
So, if we don't compile and link with xchesscc, then we need a different compiler.  What are you expecting to happen here?

Reply:
Still working in progress.  I forgot to convert it to a draft.  I was expecting it to generate llvm code and stop there.  @jackl-xilinx is this your expectation as well?

Reply:
hmm.. good idea, but that's not what aiecc.py does today, I think...  Want to look into aiecc.py to add that?

Reply:
Will do this tomorrow.

Reply:
This was added to prevent aiecc.py from trying to run with the default linker (--xbridge) which calls xchess_wrapper which then needs xchesscc which is not present for CI. I think the check for xchesscc should affect the xbridge option as well and should guide whether --xchesscc and --xbridge is on or not.